src/popups/NetzkartePopup/NetzkartePopup.js

```diff
@@ -127,7 +127,10 @@ function NetzkartePopup({ feature }) {
 
   const coordinates = transformCoords(
     feature.get('longitude') && feature.get('latitude')
-      ? [feature.get('longitude'), feature.get('latitude')]
+      ? [
+          parseFloat(feature.get('longitude'), 10),
+          parseFloat(feature.get('latitude'), 10),
+        ]
       : feature.getGeometry().getCoordinates(),
     'EPSG:21781',
```

Comment:
I think we don't need the transformation anymore.

Reply:
@oterral We still need the transformation as we can change the projection in the footer.
I have changed the data projection. The coordinates are now in EPSG:4326 instead of EPSG: 21781.

BTW: Currently, the initial projection in the popup is EPSG:4326, but the projection in the footer is EPSG:21781. I will have a look at it.